dev/sensor/htu21d/export/dev/sensor/htu21d.hpp

```diff
@@ -283,7 +283,7 @@ err htu21d<i2c_dev>::get_temperature(int &value)
     // since they contain status information
     sample &= ~3;
 
-    value = -46850 + ((175000 * static_cast<uint64_t>(sample)) / (1 << 16));
+    value = -46850 + ((175720 * static_cast<uint64_t>(sample)) / (1 << 16));
```

Comment:
I propose to:
 - substitute ` / (1 << 16) ` with only ` >> 16`
 - replace datasheet link found in https://github.com/forGGe/theCore/pull/252/files#diff-f52a45cb59ca2b566a983ba4731f3794R4 since it doesn't work anymore (pick your favorite datasheet for that)
 - give a page reference where formula can be found, if datasheet is big.